backend/Dockerfile

```diff
@@ -4,10 +4,18 @@ FROM maven:3.6.1-jdk-11-slim AS build
 ARG APPLICATION_SERVER
 ARG DATABASE_SCHEMA_HANDLING_ON_STARTUP
 ARG DATABASE_SCHEMA_CREATE_TYPE
+ARG SMTP_SERVER
```

Comment:
This should not be needed, env variables are passed automatically to dockerfile during build, see [Dockerfile in root](https://github.com/AdrianKokot/put-reunice-platform/blob/main/Dockerfile), where I removed all `ARG ...` and `ENV ...` and environment variables are still recognized properly.